### DISCUSSION TITLE
Welcome to cookieconsent Discussions!

### DISCUSSION THREAD

**Discussion by orestbida** [orestbida/cookieconsent#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ThomasFD** [comment]
Hey @orestbida, thanks for creating this cookie consent solution. I'm wondering, currently it blocks scripts, but to be GPDR compliant I also need consent for YouTube Embeds. Do you have any ideas how to implement this?

**Reply by orestbida** [reply]
Hi @ThomasFD,

for GDPR iframe management I've created an ad-hoc plugin: [iframemanager](https://github.com/orestbida/iframemanager). It's separate because there were very few requests for such a solution, and I didn't want to add additional "overhead" to the cookie consent.
